python_Docker_build/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.9
+
+RUN git clone https://github.com/tncowart/python
+
+ADD python /
+
+ENV PYTHONPATH "${PYTHONPATH}:/python"
+
+RUN pip3 install --trusted-host pypi.org pandas
```

Comment:
Is --trusted-host necessary, or does that just avoid the Zscaler issues? If it's just to get around Zscaler issue, I think you should remove it.